src/module/tests/MeleeAttackTest.ts

```diff
@@ -48,6 +49,13 @@ export class MeleeAttackTest extends SuccessTest<MeleeAttackData> {
 
         await super.prepareDocumentData();
     }
+    
+    override prepareBaseValues() {
+        if (game.settings.get(SYSTEM_NAME, FLAGS.RuleRG2TargetSizeModifiers).includes('MELEE'))
```

Comment:
In conjunction with `SuccessTest.applyTargetSizeModifiers` this rule check for both here and `RangedAttackTest` should be within that 'TargetSizeModifierFlow`, so it's never not applied should someone reuse this other places.

That way we also easier now where in code the 'target size modifier' rule is applied, as it's all using that flows 'api'.